cmd/main.go

```diff
@@ -184,3 +182,19 @@ func prepareGRPCServer(log *slog.Logger, cfg config.Config, p *proxy.GRPCProxy)
 
 	return grpcServer
 }
+
+// cors is a simple middleware that enables CORS for all requests
+func cors(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
should we make this entire section configurable?

Reply:
Mentioned this in #18 to tackle soon